alfasim_sdk/_validators.py

```diff
@@ -0,0 +1,19 @@
+def check_string_is_not_empty(instance, attribute, value):
+    """
+    A validator that raises a ValueError if the initializer is called with a empty string '' or '  '
+    """
+    if not value or value.isspace():
+        from alfasim_sdk.types import Enum
+        if type(instance) is Enum:
+            raise ValueError(f'Enum type cannot have an empty string on field "{attribute.name}"')
+
+        raise ValueError(f'The field "{attribute.name}" cannot be empty')
+
+
+def check_for_valid_unit(self, attribute, value):
```

Comment:
For consistency with the above method (and because it reads better in English), I suggest to name this `check_unit_is_valid`.